docs/svsm.md

```diff
@@ -0,0 +1,135 @@
+# SVSM  Utility Script (Launch_SVSM.sh)
+
+This 'Launch_SVSM.sh'  utility script is a provisioning and test script that performs three tasks:
+
+1. Sets up an AMD EPYC CPU powered server by building host kernel, guest kernel, IGVM,  qemu, ovmf builds
+
+2. Direct boot launch a SNP enabled guest with qemu using IGVM file
+
+3. Attest the SNP guest using the [virtee/snpguest](https://github.com/virtee/snpguest) 
+CLI tool.
+
+4. Verify measurement - igvmmeasure tool and snp attestation report
```

Comment:
```suggestion
3. Attest the SNP guest and validate the measurement using the [virtee/snpguest](https://github.com/virtee/snpguest) CLI tool and the igvmmeasure tool.
```

You did say 3 things and then enumerated 4

Reply:
Agreed. I will update.